src/app/containers/CpsRelatedContent/index.test.jsx

```diff
@@ -143,9 +128,42 @@ describe('CpsRelatedContent', () => {
     expect(getByText('February 2009', { exact: false })).not.toBeNull();
   });
   it('should render a default title if translations are not available', () => {
-    renderRelatedContentNoTitle();
+    renderRelatedContent({ pageType: STORY_PAGE, service: 'news' });
     expect(screen.getByText(`Related content`)).toBeTruthy();
   });
+
+  it('should render a translated title for Hindi Recommendations Related Content', async () => {
```

Comment:
Forgot to mention, probably a good idea to wrap the test created for optimizely in describe blocks (I followed the pattern below) like:
```
  describe.only('optimizelyExperiment', () => {
    describe('003_hindi_experiment_feature', () => {
      describe('control', () => {});
    });
  });
```
Makes it easier to teardown in my opinion.